jgrapht-core/src/test/java/org/jgrapht/alg/flow/EdmondsKarpMaximumFlowTest.java

```diff
@@ -35,27 +35,30 @@
 package org.jgrapht.alg.flow;
 
 import org.jgrapht.DirectedGraph;
+import org.jgrapht.Graph;
+import org.jgrapht.Graphs;
 import org.jgrapht.alg.interfaces.MaximumFlowAlgorithm;
 import org.jgrapht.graph.DefaultWeightedEdge;
 import org.jgrapht.graph.DirectedWeightedMultigraph;
+import org.jgrapht.graph.SimpleWeightedGraph;
 
 import java.util.Map;
 
 
 public class EdmondsKarpMaximumFlowTest extends MaximumFlowAlgorithmTestBase
 {
     @Override
-    MaximumFlowAlgorithm<Integer, DefaultWeightedEdge> createSolver(DirectedGraph<Integer, DefaultWeightedEdge> network) {
-        return new EdmondsKarpMaximumFlow<Integer, DefaultWeightedEdge>(network);
+    MaximumFlowAlgorithm<Integer, DefaultWeightedEdge> createSolver(Graph<Integer, DefaultWeightedEdge> network) {
+        return new EdmondsKarpMaximumFlow<>(network);
     }
 
     //~ Methods ----------------------------------------------------------------
 
+
     public void testCornerCases()
     {
         DirectedWeightedMultigraph<Integer, DefaultWeightedEdge> simple =
-            new DirectedWeightedMultigraph<Integer, DefaultWeightedEdge>(
-                DefaultWeightedEdge.class);
+            new DirectedWeightedMultigraph<>(DefaultWeightedEdge.class);
```

Comment:
It's better to abstract such a (notational) changes from any other (meaningful) changes. 
That'd greatly facilitate reviewing process.